float64 x 
float64 y
float64 ow 
float64 oz
int16 flag
---
int64 time
